oggm-edu/flowline_model.ipynb

```diff
@@ -88,8 +88,8 @@
     "# define horizontal resolution of the model:\n",
```

Comment:
<p>Is there a specific reason for this change? It will make the runs slower, but if there are benefits I'm happy to take the change</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OGGM/oggm-edu-notebooks/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>It was to test resolution dependence.  It doesn't need to be the default - just forgot to switch it back</p>